public/app/features/dashboard/components/SubMenu/DashboardLinksDashboard.tsx

```diff
@@ -1,77 +1,80 @@
   const styles = useStyles2(getStyles);
+  const resolvedLinks = useResolvedLinks({ dashboardUID, link });
+
+  if (!resolvedLinks || resolveLinks.length === 0) {
+    return null;
+  }
+
+  return (
+    <Menu>
+      <div className={styles.dropdown}>
+        <CustomScrollbar>
+          {resolvedLinks.map((resolvedLink, index) => {
+            return (
+              <Menu.Item
+                url={resolvedLink.url}
+                target={link.targetBlank ? '_blank' : undefined}
+                key={`dashlinks-dropdown-item-${resolvedLink.uid}-${index}`}
+                label={resolvedLink.title}
+                testId={selectors.components.DashboardLinks.link}
+                aria-label={`${resolvedLink.title} dashboard`}
+              />
+            );
+          })}
+        </CustomScrollbar>
+      </div>
+    </Menu>
+  );
+}
 
-  useLayoutEffect(() => {
-    setDropdownCssClass(getDropdownLocationCssClass(listRef.current));
-  }, [resolvedLinks]);
+export const DashboardLinksDashboard = (props: Props) => {
+  const { link, linkInfo, dashboardUID } = props;
+  const resolvedLinks = useResolvedLinks(props);
+  const styles = useStyles2(getStyles);
 
   if (link.asDropdown) {
     return (
-      <LinkElement link={link} key="dashlinks-dropdown" data-testid={selectors.components.DashboardLinks.dropDown}>
-        <>
-          <ToolbarButton
-            onClick={() => setOpened(Date.now())}
-            className={cx('gf-form-label gf-form-label--dashlink', styles.button)}
-            data-placement="bottom"
-            data-toggle="dropdown"
-            aria-expanded={!!opened}
-            aria-controls="dropdown-list"
-            aria-haspopup="menu"
-          >
-            <Icon aria-hidden name="bars" className={styles.iconMargin} />
-            <span>{linkInfo.title}</span>
-          </ToolbarButton>
-          <ul
-            id="dropdown-list"
-            className={`dropdown-menu ${styles.dropdown} ${dropdownCssClass}`}
-            role="menu"
-            ref={listRef}
-          >
-            {resolvedLinks.length > 0 &&
-              resolvedLinks.map((resolvedLink, index) => {
-                return (
-                  <li role="none" key={`dashlinks-dropdown-item-${resolvedLink.uid}-${index}`}>
-                    <a
-                      role="menuitem"
-                      href={resolvedLink.url}
-                      target={link.targetBlank ? '_blank' : undefined}
-                      rel="noreferrer"
-                      data-testid={selectors.components.DashboardLinks.link}
-                      aria-label={`${resolvedLink.title} dashboard`}
-                    >
-                      {resolvedLink.title}
-                    </a>
-                  </li>
-                );
-              })}
-          </ul>
-        </>
-      </LinkElement>
+      <Dropdown overlay={<DashboardLinksMenu link={link} dashboardUID={dashboardUID} />}>
```

Comment:
@axelavargas  - I've just fixed it [15b0fb9](https://github.com/grafana/grafana/pull/77855/commits/15b0fb951ae21c48328119c6c81aacb0e87058ed). I've used wrong component for this case (LinkButton instead of Button)